packages/expo-intent-launcher/android/src/main/java/expo/modules/intentlauncher/exceptions/PackageNotFoundException.kt

```diff
@@ -0,0 +1,10 @@
+package expo.modules.intentlauncher.exceptions
+
+import expo.modules.core.errors.CodedException
+
+class PackageNotFoundException(packageName: String) :
+  CodedException("Package not found: $packageName") {
+  override fun getCode(): String {
+    return "E_PACKAGE_NOT_FOUND"
+  }
+}
```

Comment:
```suggestion
import expo.modules.kotlin.exception.CodedException

class PackageNotFoundException(packageName: String) :
  CodedException(message = "Package not found: $packageName")
```
The `CodedException` from `.core.errors` is a legacy one.